src/n_body_simulation.py

```
@@ -307,11 +318,12 @@ def get_particle_data(self):
                   Positions are in AU, velocities in AU/(year / 2pi).
         """
         particles_data = []
+        # Build reverse mapping: index -> label
+        index_to_label = {i: l for l, i in self._label_to_index.items()}
```

Ruff (0.15.20)

[error] Ambiguous variable name: `l`

(E741)